src/service/Flags.ts

```diff
@@ -1,5 +1,6 @@
 export enum UserRegistrationStatus {
   FIRST_LOGIN = "first-login",
+  EMAIL_RESENDED = "email-resended",
```

Comment:
Desculpa a intromissão, deixei essa dica aqui mas podem ficar a vontade pra ignorar.

```suggestion
  EMAIL_RESENT = "email-resent",
```

[Conjugação do verbo resend](https://englishtivi.com/resend-v1-v2-v3-v4-v5-base-form-past-simple-past-participle-form-of-resend/#:~:text=The%20past%20tense%20of%20resend,participle%20of%20resend%20is%20resent.)